lib/plugins/aws/provider/awsProvider.js

```diff
@@ -12,6 +12,21 @@ const constants = {
 };
 
 const impl = {
+
+  /**
+   * Deprecated method. Moved to naming.js
+   * Kept here for backward compatibility
+   */
+  getStackNmae(stage) {
```

Comment:
@eahefnawy there is a typo: `Nmae` -> `Name`

wasn't it directly attached to the provided?